docs/source/notebooks/release_announcement.ipynb

```diff
@@ -0,0 +1,2785 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    sns.displot(data=df, x="probability", hue="customer_id", kind="kde", fill=True, height=3, aspect=2)
</pre><p>Shall we use a discrete <code>palette</code> like Spectral or tab10? Otherwise these ids seem to be "continuous" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Updated